doc/nvim-tree-lua.txt

```diff
@@ -414,10 +415,54 @@ Opens the tree automatically when switching tabpage or opening a new tabpage
 if the tree was previously open.
   Type: `boolean`, Default: `false`
 
+
 *nvim-tree.sort_by*
 Changes how files within the same directory are sorted.
-Can be one of 'name', 'case_sensitive', 'modification_time' or 'extension'.
-  Type: `string`, Default: `"name"`
+Can be one of 'name', 'case_sensitive', 'modification_time' or 'extension',
+'function'.
+>
+  sort_by = function(a, b)
+    if not (a and b) then
+      return true
+    end
+    if a.nodes and not b.nodes then
+      return true
+    elseif not a.nodes and b.nodes then
+      return false
+    end
+
+    return a.name:lower() <= b.name:lower()
+  end
+
+    
+  end
+  Type: `string | function(a, b)`, Default: `"name"`
+
+*nvim-tree.after_sort*
+Related to nvim-tree.sort_by, this function runs without mergesort.
```

Comment:
Why do we need this? Is it because mixin sort cannot be achieved via a comparator?